### PR TITLE
Js remove pokemon from team

### PR DESCRIPTION
**Pull request by jordy1611**
### Type of change made:
- [ ] Bugfix (non-breaking change which fixes an issue)
- [x] New feature (non-breaking change which adds functionality)
- [ ] Breaking change (fix or feature that would cause existing functionality to not work as expected)
- [ ] Testing - building tests
- [ ] Formatting - no new features
- [ ] Styling- no new features
### Detailed Description (Why is this change required? What problem does it solve? Any challenges that arose while implementing this feature? If so, how were the addressed?)
Added feature to track user's team in app
Create functionality to remove pokemon from team
### Files modified:
- [ ] index.html
- [x] App
- [ ] CardContainer
- [ ] PokeCard
- [x] PokeForm
- [ ] Login
- [x] FormContainer
- [ ] index.js
- [ ] Other files:

### CODE REVIEW

File: stretch-auth/src/FormContainer/FormContainer.js [hunk 1]
```diff
@@ -38,13 +38,29 @@ class FormContainer extends Component {
...
+  updateUserPokeTeam() {
+    const userPokemon = {
+      1: this.state.slot1,
+      2: this.state.slot2,
+      3: this.state.slot3,
+      4: this.state.slot4,
+      5: this.state.slot5,
+      6: this.state.slot6,
+     }
+    this.props.updateUserPokeTeam(userPokemon)
```
Comment (jordy1611) [inline stretch-auth/src/FormContainer/FormContainer.js:54]: I would setState for user team here

**Review by jordy1611 — COMMENTED** [review]
I would get Item userTeam from localStorage on componentDidMount and just do a setState on userPokeTeam

**Review by H-Bo214 — APPROVED** [review]
Great work getting the delete functionality working.

**Review by AurumValian — APPROVED** [review]
Thanks for the comments, I'll get working on that

**Pull request merged**